cmd/operator-sdk/bundle/create.go

```diff
@@ -46,65 +46,94 @@ func newCreateCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "create",
 		Short: "Create an operator bundle image",
-		Long: `The 'operator-sdk bundle create' command will build an operator
-bundle image containing operator metadata and manifests, tagged with the
-provided image tag.
-
-To write all files required to build a bundle image without building the
-image, set '--generate-only=true'. A bundle Dockerfile, bundle metadata, and
-a 'manifests/' directory containing your bundle manifests will be written if
-'--generate-only=true':
-
-	$ operator-sdk bundle create --generate-only --directory ./deploy/olm-catalog/test-operator/0.1.0
-	$ ls .
-	...
-	bundle.Dockerfile
-	...
-	$ tree ./deploy/olm-catalog/test-operator/
-	└── 0.1.0
-		└── example.com_tests_crd.yaml
-		└── test-operator.v0.1.0.clusterserviceversion.yaml
-	└── manifests
-		└── example.com_tests_crd.yaml
-		└── test-operator.v0.1.0.clusterserviceversion.yaml
-	└── metadata
-		└── annotations.yaml
-
-'--generate-only' is useful if you want to build an operator's bundle image
-manually, modify metadata before building an image, or want to generate a
-'manifests/' directory containing your operator manifests for compatibility
+		Long: `
+The 'operator-sdk bundle create' command will build an operator bundle image containing operator metadata and 
+manifests, tagged with the provided image tag. 
+		
+To write all files required to build a bundle image without building the image, set '--generate-only=true'. 
+		
+Notes:
+* The /manifests folder is expected to contain resources that can be applied to the cluster using 
+standard tooling like kubectl.
+* The /metadata folder is expected to contain resources that are not directly apply able. It can be used to store 
+supporting metadata associated with the operator.
+* The image is not runnable, it is built from scratch.
+* The '--generate-only' flag is useful if you want to build an operator's bundle image manually, modify metadata before 
+building an image, or want to generate a 'manifests/' directory containing your operator manifests for compatibility
 with other operator tooling.
 
-More information on operator bundle images and metadata:
-https://github.com/openshift/enhancements/blob/master/enhancements/olm/operator-bundle.md#docker
-
-NOTE: bundle images are not runnable.
+For more information on operator bundle images and metadata format see: 
+		https://github.com/operator-framework/operator-registry#manifest-format 
 `,
-		Example: `The following invocation will build a test-operator 0.1.0 bundle image using Docker.
-This image will contain manifests for package channels 'stable' and 'beta':
+		Example: `
+
+The following invocation will build a memcached-operator bundle metadata with '--generate-only'.
+		
+  # The following is the directory of an operator registry bundle generated 
+	
+  $ operator-sdk bundle create --generate-only
```

Comment:
This isn't an example of running the command, just a description of what is generated when using `--generate-only`.

Reply:
I agree. It was improved.